data/widgets/export_sprite_sheet.xml

```diff
@@ -17,7 +17,14 @@
     <hbox />
     <vbox>
       <check id="padding_enabled" text="@.padding" />
-      <check id="trim_enabled" text="@.trim" />
+      <check id="trim_enabled" text="@.trim" cell_align="left"/>
+      <grid columns="2" id="trim_container" cell_hspan="1">
+        <hbox>
+          <boxfiller />
+          <boxfiller />
+            <check id="grid_trim_enabled" text="@.grid" cell_align="right"/>
+        </hbox>
+        </grid>
```

Comment:
Several changes here are not required (like two consecutive `<boxfiller />`s, or the `cell_align="left"` which does nothing because `cell_*` attributes are useful only when the direct parent XML element is a `<grid>`).

You can replace this with:
```xml
    <vbox>
      <check id="padding_enabled" text="@.padding" />
      <check id="trim_enabled" text="@.trim" />
      <hbox id="trim_container">
        <boxfiller />
        <check id="grid_trim_enabled" text="@.trim_by_grid" />
      </hbox>
      <check id="extrude_enabled" text="@.extrude" />
    </vbox>
```
Removing the `<grid>` (And using a `<hbox>` as the `trim_container`)

Reply:
Done